Find-and-Replace/Find-match-case/.NET/Find-match-case/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIO;
+using Syncfusion.Drawing;
+
+using (FileStream fileStreamPath = new FileStream(Path.GetFullPath(@"Data/Template.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+{
+    //Opens an existing Word document.
+    using (WordDocument document = new WordDocument(fileStreamPath, FormatType.Automatic))
+    {
+        //Finds the first occurrence of a particular text in the document.
```

Comment:
// Finds the first occurrence of a particular text that matches the exact case in the document.